mvcc/kvstore.go

```diff
@@ -44,6 +45,8 @@ var (
 	ErrClosed    = errors.New("mvcc: closed")
 
 	plog = capnslog.NewPackageLogger("github.com/coreos/etcd", "mvcc")
+
+	EmptyKeep = make(map[revision]struct{})
```

Comment:
probably can be now used for `make(struct map[rev]struct{}` in your comment below

"If there are no scheduled compactions, won't this always force a commit to disk? Isn't that something keep is trying to avoid? a possible workaround is to set s.keep = make(struct map[rev]struct{}) after this ForceCommit (and have len(keep) > 0 instead of keep != nil in UnsafeForEach) so it won't commit on every call."